components/organisms/Hero/Hero.js

```diff
@@ -11,6 +11,7 @@ import styles from './Hero.module.css'
  * @param {string} props.backgroundImage The background image object.
  * @param {string} props.body            Text for the body.
  * @param {string} props.className       The className.
+ * @param {any} props.children           InnerBlocks.
```

Comment:
```suggestion
 * @param {any}    props.children           InnerBlocks.
```

Fixes alignment of JSDoc.

Reply:
@gregrickaby Alignment Fix.